src/components/Match/matchColumns.jsx

```diff
@@ -770,7 +770,7 @@ export const teamfightColumns = [
     displayFn: inflictorRow(items),
   },
 ];
-
+/*
 const purchaseObserverColumn = {
```

Comment:
To make the tables fit better I removed the purchase columns (since this page is more about actual usage).

Purchase data remains on the purchase page.